src/Microsoft.IdentityModel.Protocols.WsFederation/WsFederationMessage.cs

```diff
@@ -181,13 +181,20 @@ public virtual string GetToken()
                 XmlReader xmlReader = XmlReader.Create(sr);
                 xmlReader.MoveToContent();
 
-                XmlUtil.CheckReaderOnEntry(xmlReader, WsTrustConstants.Elements.RequestSecurityTokenResponse, WsTrustConstants.Namespaces.WsTrust2005);
+                var namespaceList = new List<string>()
```

Comment:
Probably want to move these somewhere and make them static so we don't allocate them everytime.